test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/compute.tf

```diff
@@ -0,0 +1,33 @@
+data "google_compute_image" "default" {
+  # https://cloud.google.com/compute/docs/images
+  family  = "ubuntu-minimal-2004-lts"
+  project = "ubuntu-os-cloud"
+}
+
+resource "google_compute_instance" "default" {
+  name = "test-${var.TEST_RUN_ID}"
```

Comment:
Let's add a prefix identifying the instance with elastic-package. The `test` prefix may be too vague.